br/pkg/task/common.go

```diff
@@ -72,7 +76,14 @@ const (
 	defaultGRPCKeepaliveTime    = 10 * time.Second
 	defaultGRPCKeepaliveTimeout = 3 * time.Second
 
-	unlimited = 0
+	flagCrypterType       = "crypter.method"
+	flagCipherKey         = "crypter.key"
+	flagCipherKeyFilePath = "crypter.key-filepath"
```

Comment:
choose to stick with either "cipher" or "crypter" in naming `flagXXXX`, don't mix the two.